packages/x-ray/plugin-android-screenshots/src/plugin.ts

```diff
@@ -47,8 +48,8 @@ export const androidScreenshots = (options?: TAndroidScreenshotsOptions): TPlugi
       appName: 'X-Ray',
       appId: 'org.nextools.xray',
       entryPointPath,
-      fontsDir: options?.fontsDir,
-      dependencyNames: [
+      fontsDir: opts?.fontsDir,
+      dependencyNames: opts?.dependencyNames ?? [
```

Comment:
`react-native-view-shot` dependency is mandatory and crucial for native x-ray to work, so it can't be skipped.

On the other hand `react-native-svg` is optional, but making it so would lead to a breaking change.

Let's agree on something like this:

```ts
const opts = {
  shouldBailout: false,
  dependencyNames: [],
  ...options,
}

// ...

dependencyNames:  [
  'react-native-view-shot',
  'react-native-svg',
  ...opts.dependencyNames
],
```

Reply:
Thanks! I applied the changes you proposed both in android and ios. Then I testPublish it and tested both on android and ios in my project (with and without adding more dependencies). Works great 👍